### PR TITLE
Docs/ correcting attribute section byline

### PR DESCRIPTION
**Pull request by philnichol**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #15842
Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
NONE
```

Replaced incorrect "Attributes Reference" bylines since there were a lot of them.
@bflad  I purposefully ignored either resources that have no additional attribute references (eg. This resource does not export any additional attributes.) , or ones that had restricted attribute references (eg. All of the argument attributes except auto_accept are also exported as result attributes.). Wasn't sure how you wanted those handled?

Before:
```
Error checking Terraform Provider documentation: 125 errors occurred:
        * website/docs/r/acm_certificate.html.markdown: error checking file contents: import section heading level (3) should be: 2
        * website/docs/r/api_gateway_base_path_mapping.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_gateway_response.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_integration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_integration_response.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method_response.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method_settings.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/appautoscaling_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/appautoscaling_target.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_lifecycle_hook.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/autoscaling_schedule.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/batch_compute_environment.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/cloudwatch_event_target.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cloudwatch_log_destination_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cloudwatch_log_subscription_filter.html.markdown: error checking file contents: attributes section byline (No extra attributes are exported.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/codecommit_trigger.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cognito_identity_provider.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cognito_user_group.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/config_configuration_recorder_status.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/config_remediation_configuration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cur_report_definition.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/db_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/default_network_acl.html.markdown: error checking file contents: example section heading (Example config to deny all Egress traffic, allowing Ingress) should be: Example Usage
        * website/docs/r/default_route_table.html.markdown: error checking file contents: example section heading (Example usage with tags) should be: Example Usage
        * website/docs/r/default_security_group.html.markdown: error checking file contents: example section heading (Example config to deny all Egress traffic, allowing Ingress) should be: Example Usage
        * website/docs/r/devicefarm_project.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/directory_service_conditional_forwarder.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/directory_service_log_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/dms_event_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/dx_connection_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ebs_default_kms_key.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ebs_encryption_by_default.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ec2_client_vpn_authorization_rule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ecs_task_definition.html.markdown: error checking file contents: example section code block language (json) should be: ```hcl or ```terraform
        * website/docs/r/eip_association.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/elasticsearch_domain_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elastictranscoder_pipeline.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elastictranscoder_preset.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elb_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/emr_instance_fleet.html.markdown: error checking file contents: title section heading (aws_emr_instance_fleet) should have prefix: "Data Source: " or "Resource: "
        * website/docs/r/emr_managed_scaling_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/glue_dev_endpoint.markdown: error checking file contents: title section heading (aws_glue_dev_endpoint) should have prefix: "Data Source: " or "Resource: "
        * website/docs/r/glue_resource_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_account_alias.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_group_membership.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_group_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_group_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_instance_profile.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_role_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_role_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_server_certificate.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_user_group_membership.html.markdown: error checking file contents: example section heading (Example usage) should be: Example Usage
        * website/docs/r/iam_user_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/iam_user_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iot_policy_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iot_thing_principal_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/kinesis_firehose_delivery_stream.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/kinesis_stream.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/kinesis_video_stream.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/lambda_alias.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/lambda_event_source_mapping.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/lambda_function.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/lambda_layer_version.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/lambda_permission.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/lb_listener_certificate.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/lightsail_domain.html.markdown: error checking file contents: example section heading (Example Usage, creating a new domain) should be: Example Usage
        * website/docs/r/lightsail_key_pair.html.markdown: error checking file contents: example section heading (Example Usage, creating a new Key Pair) should be: Example Usage
        * website/docs/r/load_balancer_backend_server_policy.html.markdown: error checking file contents: example section code block language (shell) should be: ```hcl or ```terraform
        * website/docs/r/load_balancer_listener_policy.html.markdown: error checking file contents: example section heading (Example Usage for Custom Policy) should be: Example Usage
        * website/docs/r/load_balancer_policy.html.markdown: error checking file contents: example section code block language (shell) should be: ```hcl or ```terraform
        * website/docs/r/media_store_container_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/neptune_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/network_interface_sg_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/organizations_policy.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/organizations_policy_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_adm_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_sandbox_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_voip_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_voip_sandbox_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_baidu_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_event_stream.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_gcm_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/quicksight_user.html.markdown: error checking file contents: attributes section byline (All above attributes except for session_name and identity_type are exported as well as:) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/redshift_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/redshift_snapshot_copy_grant.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/redshift_snapshot_schedule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/redshift_snapshot_schedule_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_analytics_configuration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_inventory.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_metric.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_notification.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/secretsmanager_secret.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/secretsmanager_secret_policy.html.markdown: error checking file contents: attributes section heading (Attribute Reference) should be: Attributes Reference
        * website/docs/r/secretsmanager_secret_rotation.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/secretsmanager_secret_version.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/ses_active_receipt_rule_set.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_configuration_set.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_event_destination.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_identity_notification_topic.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_identity_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_receipt_rule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_receipt_rule_set.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sns_sms_preferences.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sns_topic_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/spot_datafeed_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sqs_queue_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ssm_maintenance_window_target.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/ssm_resource_data_sync.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/storagegateway_stored_iscsi_volume.html.markdown: error checking file contents: import section code block text should contain resource name: aws_storagegateway_stored_iscsi_volume
        * website/docs/r/storagegateway_tape_pool.html.markdown: error checking file contents: attributes section heading (Attribute Reference) should be: Attributes Reference
        * website/docs/r/transfer_server.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/transfer_ssh_key.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/transfer_user.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/volume_attachment.html.markdown: error checking file contents: attributes section byline should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/vpc_peering_connection_accepter.html.markdown: error checking file contents: attributes section byline (All of the argument attributes except auto_accept are also exported as result attributes.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/vpc_peering_connection_options.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/vpn_gateway_route_propagation.html.markdown: error checking file contents: attributes section byline (This resource does not export any additional attributes.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/wafregional_web_acl_association.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/wafv2_web_acl_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/workspaces_directory.html.markdown: error checking file contents: arguments section heading (Arguments Reference) should be: Argument Reference
```

After:
```
$ tfproviderdocs check -allowed-resource-subcategories-file website/allowed-subcategories.txt -provider-name aws -require-resource-subcategory -enable-contents-check
Error checking Terraform Provider documentation: 102 errors occurred:
        * website/docs/r/acm_certificate.html.markdown: error checking file contents: import section heading level (3) should be: 2
        * website/docs/r/api_gateway_base_path_mapping.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_gateway_response.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_integration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_integration_response.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method_response.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/api_gateway_method_settings.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/appautoscaling_target.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_lifecycle_hook.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/autoscaling_policy.html.markdown: error checking file contents: missing arguments section: ## Argument Reference
        * website/docs/r/cloudwatch_event_target.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cloudwatch_log_destination_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cloudwatch_log_subscription_filter.html.markdown: error checking file contents: attributes section byline (No extra attributes are exported.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/codecommit_trigger.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cognito_identity_provider.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cognito_user_group.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/config_configuration_recorder_status.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/config_remediation_configuration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/cur_report_definition.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/db_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/default_network_acl.html.markdown: error checking file contents: example section heading (Example config to deny all Egress traffic, allowing Ingress) should be: Example Usage
        * website/docs/r/default_route_table.html.markdown: error checking file contents: example section heading (Example usage with tags) should be: Example Usage
        * website/docs/r/default_security_group.html.markdown: error checking file contents: example section heading (Example config to deny all Egress traffic, allowing Ingress) should be: Example Usage
        * website/docs/r/devicefarm_project.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/directory_service_conditional_forwarder.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/directory_service_log_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/dms_event_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/dx_connection_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ebs_default_kms_key.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ebs_encryption_by_default.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ec2_client_vpn_authorization_rule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ecs_task_definition.html.markdown: error checking file contents: example section code block language (json) should be: ```hcl or ```terraform
        * website/docs/r/elasticsearch_domain_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elastictranscoder_pipeline.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elastictranscoder_preset.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/elb_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/emr_instance_fleet.html.markdown: error checking file contents: title section heading (aws_emr_instance_fleet) should have prefix: "Data Source: " or "Resource: "
        * website/docs/r/emr_managed_scaling_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/glue_dev_endpoint.markdown: error checking file contents: title section heading (aws_glue_dev_endpoint) should have prefix: "Data Source: " or "Resource: "
        * website/docs/r/glue_resource_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_account_alias.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_group_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_role_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iam_user_group_membership.html.markdown: error checking file contents: example section heading (Example usage) should be: Example Usage
        * website/docs/r/iam_user_policy_attachment.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iot_policy_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/iot_thing_principal_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/lambda_permission.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/lb_listener_certificate.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/lightsail_domain.html.markdown: error checking file contents: example section heading (Example Usage, creating a new domain) should be: Example Usage
        * website/docs/r/lightsail_key_pair.html.markdown: error checking file contents: example section heading (Example Usage, creating a new Key Pair) should be: Example Usage
        * website/docs/r/load_balancer_backend_server_policy.html.markdown: error checking file contents: example section code block language (shell) should be: ```hcl or ```terraform
        * website/docs/r/load_balancer_listener_policy.html.markdown: error checking file contents: example section heading (Example Usage for Custom Policy) should be: Example Usage
        * website/docs/r/load_balancer_policy.html.markdown: error checking file contents: example section code block language (shell) should be: ```hcl or ```terraform
        * website/docs/r/media_store_container_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/neptune_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/network_interface_sg_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/organizations_policy_attachment.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_adm_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_sandbox_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_voip_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_apns_voip_sandbox_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_baidu_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_event_stream.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/pinpoint_gcm_channel.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/quicksight_user.html.markdown: error checking file contents: attributes section byline (All above attributes except for session_name and identity_type are exported as well as:) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/redshift_event_subscription.html.markdown: error checking file contents: attributes section heading (Attributes) should be: Attributes Reference
        * website/docs/r/redshift_snapshot_schedule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/redshift_snapshot_schedule_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_analytics_configuration.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_inventory.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_metric.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_notification.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/s3_bucket_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/secretsmanager_secret_policy.html.markdown: error checking file contents: attributes section heading (Attribute Reference) should be: Attributes Reference
        * website/docs/r/ses_active_receipt_rule_set.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_configuration_set.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_event_destination.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_identity_notification_topic.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_identity_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_receipt_rule.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ses_receipt_rule_set.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sns_sms_preferences.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sns_topic_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/spot_datafeed_subscription.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/sqs_queue_policy.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/ssm_maintenance_window_target.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/ssm_resource_data_sync.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/storagegateway_stored_iscsi_volume.html.markdown: error checking file contents: import section code block text should contain resource name: aws_storagegateway_stored_iscsi_volume
        * website/docs/r/storagegateway_tape_pool.html.markdown: error checking file contents: attributes section heading (Attribute Reference) should be: Attributes Reference
        * website/docs/r/transfer_server.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/transfer_ssh_key.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/transfer_user.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/vpc_peering_connection_accepter.html.markdown: error checking file contents: attributes section byline (All of the argument attributes except auto_accept are also exported as result attributes.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/vpc_peering_connection_options.html.markdown: error checking file contents: title section code examples should be in Example Usage section
        * website/docs/r/vpn_gateway_route_propagation.html.markdown: error checking file contents: attributes section byline (This resource does not export any additional attributes.) should be: In addition to all arguments above, the following attributes are exported:
        * website/docs/r/wafregional_web_acl_association.html.markdown: error checking file contents: missing example section: ## Example Usage
        * website/docs/r/wafv2_web_acl_association.html.markdown: error checking file contents: missing attributes section: ## Attributes Reference
        * website/docs/r/workspaces_directory.html.markdown: error checking file contents: arguments section heading (Arguments Reference) should be: Argument Reference

```

### CODE REVIEW

File: website/docs/r/autoscaling_policy.html.markdown [hunk 1]
```diff
@@ -39,9 +39,7 @@ resource "aws_autoscaling_group" "bar" {
 }
 ```
 
-## Argument Reference
-
-The following arguments are supported:
+## Arguments Reference
```
Comment (bflad) [inline website/docs/r/autoscaling_policy.html.markdown:42]: Should keep the old (singular) header here -- the plurality discrepancy between the two headers is an unfortunate historical one 😅 

```
* website/docs/r/autoscaling_policy.html.markdown: error checking file contents: missing arguments section: ## Argument Reference
```

```suggestion
## Argument Reference
```

**Review by bflad — COMMENTED** [review]
Overall we're a little leery of broad codebase pull requests like this since they have a larger potential to introduce merge conflicts, but this instance should be okay since that documentation header, byline, and first list item should be fairly untouched in most pull requests. 👍  Thank you for adjusting these. Once the one case below is fixed we can get this in.

> @bflad I purposefully ignored either resources that have no additional attribute references (eg. This resource does not export any additional attributes.) , or ones that had restricted attribute references (eg. All of the argument attributes except auto_accept are also exported as result attributes.). Wasn't sure how you wanted those handled?

In most cases we should add the header, byline, and include a description for the current `id` (`d.SetId()`) attribute, since every resource and data source inherently must include this due to Terraform Plugin SDK requirements. If we find cases where there are absolutely no new attributes (e.g. a data source that includes an `id` argument), then we can pick a standard phrasing for the linter to succeed or add file ignores to the linter.

**Comment by bflad** [comment]
Oh I should mention that those header, byline, `id` attribute additions should be a separate PR please. 😄

**Comment by philnichol** [comment]
@bflad  cheers for the feedback! Good point on the massive PR, I'll split things into less files per PR in future!

**Pull request merged**